packages/appstore/src/components/modals/real-wallets-upgrade/components/modal-elements/wallets-upgrade-footer/wallets-upgrade-footer.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { Button, Modal, ProgressBarTracker } from '@deriv/components';
+import { useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import WalletSteps from '../wallet_steps';
+import { TRealWalletsUpgradeSteps } from 'Types';
+import './wallets-upgrade-footer.scss';
+
+type TWhyWalletsFooter = {
+    handleClose: VoidFunction;
+    handleNext: VoidFunction;
+};
+
+type TReadyToEnableWalletsFooter = {
+    handleBack: VoidFunction;
+    is_disabled: boolean;
+    upgradeToWallets: (value: boolean) => void;
+};
+
+export const WhyWalletsFooter = ({ handleNext }: TWhyWalletsFooter) => {
```

Comment:
Can we move WhyWalletsFooter and ReadyToEnableWalletsFooter as separate components to WhyWallets and ReadyToEnableWallets folders respectively?

Reply:
Will do this, this would make a simpler structure

Reply:
Done